frontend/viewdir/capabilities.njk

```diff
@@ -0,0 +1,13 @@
+{% extends "template.njk" %}
+
+{% block content %}
+    <div class="container">
+        <h5 style="color:#575756"><spam id="capTitle">Capabilities</span></h5>
+        <div class="divider"></div>
+        {% for item in items %}
+            <div class="section">
+                <h5 style="color:#009fe3"><a id="capName" href="/capabilities/{{item.capabilityID}}">{{item.capability}}</a></h5>
+            </div>
+        {% endfor %}
```

Comment:
Colours should be in CSS files, not the html.
Also you've typo'd `span` as `spam`